packages/eventual-send/package.json

```
@@ -2,12 +2,11 @@
   "name": "@agoric/eventual-send",
   "version": "0.13.22",
   "description": "Extend a Promise class to implement the eventual-send API",
-  "parsers": {
-    "js": "mjs"
-  },
+  "type": "module",
   "main": "src/no-shim.js",
   "types": "src/index.d.ts",
   "scripts": {
+    "cover": "c8 ava",
```

This script needs a specific name to be driven from the top level and ci coverage publishing.

```suggestion
    "test:c8": "c8 ava",
```

---

Integrated this, and also the options you describe in COVERAGE.md. Thanks for the detailed notes.